apps/posts/src/views/PostAnalytics/PostAnalytics.tsx

```
@@ -60,34 +60,34 @@
-                                <BreadcrumbPage>
+                                <BreadcrumbPage className='leading-[24px]'>
                                 Web
                                 </BreadcrumbPage>
                             </BreadcrumbItem>
                         </BreadcrumbList>
                     </Breadcrumb>
-                    <H1 className='mt-0.5 min-h-[35px] indent-0'>
+                    <H1 className='min-h-[35px] indent-0 leading-[1.2em]'>
                         {post && post.title}
                     </H1>
                     {typedPost && typedPost.published_at && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)